saskatoon/harvest/forms.py

```diff
@@ -441,12 +455,35 @@ class Meta:
         required=False,
     )
 
-    def __init__(self, *args, **kwargs):
-        if 'yields' in kwargs:
+    equipment_point = forms.ModelChoiceField(
+        queryset=Organization.objects.all(),
+        widget=autocomplete.ModelSelect2(
+            url='equipmentpoint-autocomplete',
+            forward=['id', 'start_date', 'end_date'],
+        ),
+        label=_("Equipment Point"),
+        required=False,
+    )
+
+    def __init__(self, *args, **kwargs) -> None:
+        if "yields" in kwargs:
             self.yields = kwargs.pop('yields')
+
+        instance = kwargs.get('instance', None)
         super().__init__(*args, **kwargs)
 
-    def clean_end_date(self):
+        if instance is not None and hasattr(instance, 'id'):
+            self.initial['id'] = instance.id  # type: ignore
+
+        # Assumes that a harvest can only reserve one equipment
+        # point at a time so all reserved equipment belongs to the same owner.
+        if instance is not None and hasattr(instance, 'equipment_reserved'):
```

Comment:
Q: in which circumstances would an `instance` be not None but have no 'equipment_reserved' field ?